datameta/security/__init__.py

```diff
@@ -309,3 +303,27 @@ def revalidate_admin(request):
     if user.site_admin or user.group_admin:
         return user
     raise HTTPUnauthorized()
+
+
+def successful_authenticated(**kwargs) -> None:
```

Comment:
this should be named `successfully_authenticated` or `successful_authentication` 🇬🇧 (sorry :P)

Reply:
oh true, thanks! :D